add win-arm64 and rerender

ruben-arts opened the pull request:
<!--
Thank you for your pull request.
Below are a few things we ask you kindly to self-check before getting a review. Remove checks that are not relevant.
-->
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

<!--
Please note any issues this fixes using [closing keywords]( https://help.github.com/articles/closing-issues-using-keywords/ ):
-->

<!--
Please add any other relevant info below:
-->
This PR is a pure guess, I've not moved any projects to `win-arm64` yet so pointers are welcome!

I would like this to be available so that we can also package `pixi` for `win-arm64` because it uses this as a dependency.

ruben-arts committed:
add win-arm64 and rerender

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe/recipe.yaml```) and found it was in an excellent condition.

I do have some suggestions for making it better though...


For **recipe/recipe.yaml**:

 * ℹ️ The feedstock is lowering the image versions for one or more platforms: {'linux_64': 'cos7'} (the default is alma9). Unless you are in the very rare case of repackaging binary artifacts, consider removing these overrides from conda-forge.yml in the top feedstock directory.


<sub>This message was generated by GitHub Actions workflow run [https://github.com/conda-forge/conda-forge-webservices/actions/runs/16562738079](https://github.com/conda-forge/conda-forge-webservices/actions/runs/16562738079). Examine the logs at this URL for more detail.</sub>

xhochy committed:
MNT: Re-rendered with conda-smithy 3.56.3 and conda-forge-pinning 2026.03.14.04.43.55

Other tools:
- conda-build 26.1.0
- rattler-build 0.58.4
- rattler-build-conda-compat 1.4.11

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe/recipe.yaml```) and found it was in an excellent condition.

xhochy commented:
> Cannot solve the request because of: No
  │ candidates were found for msinttypes *
